resmon

```diff
@@ -77,7 +79,15 @@ unless($debug) {
     open(STDIN, "</dev/null");
     open(STDOUT, ">/dev/null");
     open(STDERR, ">/dev/null");
-    fork && exit;
+    my $pid=fork;
+    if ($pid) {
+      if (my $PIDFILE=$config->{pidfile}) {
+        open(PIDFILE,">$PIDFILE") or die "can't open >$PIDFILE";
+        print PIDFILE $pid;
+        close PIDFILE or die "can't close $PIDFILE";
+      }
+      exit;
+    }
```

Comment:
Could fix style (spaces around operators and commas) if that matters.

Not sure about convention here, but maybe lowercase the `$PIDFILE` var so it doesn't look like the file handle `PIDFILE`, or just keep using `$config->{pidfile}`.